front/app/containers/Admin/reporting/components/ReportBuilder/Toolbox/index.tsx

```diff
@@ -94,7 +96,7 @@ const ReportBuilderToolbox = ({ reportId }: ReportBuilderToolboxProps) => {
           <DraggableElement
             id="e2e-draggable-about-report"
             component={
-              <AboutReportWidget reportId={reportId} projectId={undefined} />
+              <AboutReportWidget reportId={reportId} projectId={projectId} />
```

Comment:
When you're in a phase report context, all these widgets will now be initialized with the correct project id. Still have to see if we can also initialize them with a smart phase id

Reply:
Hmm partly, I still need to add the 'All projects' option for the project select. But I will do it after the migration. Think you can close that one for now